soundbay/conf/runs/inference_single_audio_manatees.yaml

```diff
@@ -1,5 +1,5 @@
 # @package _global_
 defaults:
   - data: inference_file_manatees
-  - checkpoint: defaults
+  - checkpoint: inference
```

Comment:
You discarded the "checkpoint" section of the hierarchy, didn't you? Is it still supposed to be here?